src/components/VariantSearch.tsx

```diff
@@ -4,18 +4,34 @@ import React, { useState } from 'react';
 import { VariantSearchField } from './VariantSearchField';
 import { AnalysisMode } from '../data/AnalysisMode';
 
+type SearchType = 'aa-mutation' | 'nuc-mutation' | 'pango-lineage';
+
+type SearchOption = {
+  label: string;
+  value: string;
+  type: SearchType;
+};
+
 type Props = {
   currentSelection?: VariantSelector[];
   onVariantSelect: (selections: VariantSelector[], analysisMode?: AnalysisMode) => void;
   analysisMode: AnalysisMode;
+  selectedOptions1?: SearchOption[];
+  variantQuery1?: string;
```

Comment:
- What is the purpose of the two new variables? Which information do they carry that is not already in `currentSelection`?
- Why is a `1` at the end?